.github/workflows/check-changelog.yml

```diff
@@ -0,0 +1,45 @@
+name: changelog
+
+on:
+  pull_request:
+    types: [opened, synchronize, labeled, unlabeled]
+
+jobs:
+  check:
+    name: Check changelog entry
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/github-script@v6
+        id: check
+        with:
+          script: |
+            const fs = require('node:fs')
+            const path = require('node:path')
+          
+            const { data: { labels: labels } } = await github.rest.pulls.get({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              pull_number: context.issue.number
+            })
+            const labelNames = labels.map(label => label.name)
+
+            if (labelNames.includes('skip-changelog')) {
```

Comment:
`skip-changelog` label can be used to skip changelog check.

Reply:
Do we actually need a way to skip the changelog? It looks like failing it lets the other workflows run anyway.

Reply:
I was thinking similarly, I decided it's nice to have this label so that we can indicate that the PR didn't need a CHANGELOG and get a green CI tick (if author wants a green truck).

Also then users can search on the label to see all undocumented changes.

Reply:
Addressed.